.github/workflows/deployToTest.yml

```diff
@@ -39,3 +42,29 @@ jobs:
         with:
           dockerfile: 'run_gcs_to_bq/Dockerfile'
           image-path: 'gcr.io/${{ secrets.TEST_PROJECT_ID }}/gcs-to-bq'
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        working-directory: ./config
+      - name: Terraform Init
+        working-directory: ./config
+        run: terraform init
+      - name: Terraform Format
+        working-directory: ./config
+        run: terraform fmt -check
+      - name: Terraform Plan
```

Comment:
Is there benefit to running terraform plan here if it's just going to auto approve? Does this give any information to human users?

Reply:
terraform plan was included in the example starter workflow. I assumed it was necessary to refresh the state before apply, but reading the documentation more closely, it looks like terraform refreshes the state before any command. I will remove it.